modules/monitoring/prometheus/main.tf

```diff
@@ -141,22 +141,28 @@ locals {
   conf       = merge(local.conf_defaults, var.conf)
   password   = var.grafana_password == "" ? random_password.grafana_password.result : var.grafana_password
   conf_defaults = {
-    "alertmanager.enabled"       = true
-    "grafana.enabled"            = true
-    "grafana.pvc.enabled"        = true
-    "grafana.ingress.enabled"    = true
-    "grafana.ingress.hosts[0]"   = "grafana.${var.domains[0]}"
-    "grafana.adminPassword"      = local.argocd_enabled > 0 ? "KMS_ENC:${aws_kms_ciphertext.grafana_password[0].ciphertext_blob}:" : local.password
-    "grafana.google.auth"        = var.grafana_google_auth
-    "grafana.allowed.domains"    = var.grafana_allowed_domains
-    "prometheus.enabled"         = true
-    "prometheus.ingress.enabled" = false
-    "namespace"                  = local.namespace
-    "rbac.create"                = true,
-    "resources.limits.cpu"       = "100m",
-    "resources.limits.memory"    = "300Mi",
-    "resources.requests.cpu"     = "100m",
-    "resources.requests.memory"  = "300Mi"
+    "alertmanager.enabled"                                        = true
+    "grafana.enabled"                                             = true
+    "grafana.pvc.enabled"                                         = true
+    "grafana.ingress.enabled"                                     = true
+    "grafana.ingress.hosts[0]"                                    = "grafana.${var.domains[0]}"
```

Comment:
what if we want to pass empty `var.domains` and access grafana via port-forwarding? seems like we will face index error here, what do you think?

Reply:
we have a local domain by default, it's not empty